packages/client-utils/src/sync-tools.js

```diff
@@ -287,3 +287,101 @@ export const waitUntilInvitationReceived = (addr, io, options) => {
     { reusePromise: true, setTimeout, ...resolvedOptions },
   );
 };
+
+/// ////////// Make sure an election held by a given committee (see @agoric/governance) turned out as expected /////////////
+
+/**
+ * @typedef {{
+ *   latestOutcome: {
+ *     outcome: string;
+ *     question: import('@endo/marshal').RemotableObject
+ *   },
+ *   latestQuestion: {
+ *     closingRule: { deadline: bigint },
+ *     questionHandle: import('@endo/marshal').RemotableObject
+ *   }
+ * }} ElectionResult
+ */
+
+/**
+ * @param {string} basePath
+ * @param {{follow: (...params: string[]) => Promise<object>; marshaller: import("@endo/marshal").Marshal<any>}} io
+ * @returns {Promise<ElectionResult>}
+ */
+export const fetchLatestEcQuestion = async (basePath, io) => {
+  const { follow, marshaller } = io;
+  const pathOutcome = `:${basePath}.latestOutcome`;
+  const pathQuestion = `:${basePath}.latestQuestion`;
+
+  const [latestOutcome, latestQuestion] = await Promise.all([
+    follow('-lF', pathOutcome, '-o', 'text').then(outcomeRaw =>
+      marshaller.fromCapData(JSON.parse(outcomeRaw)),
+    ),
+    follow('-lF', pathQuestion, '-o', 'text').then(questionRaw =>
+      marshaller.fromCapData(JSON.parse(questionRaw)),
+    ),
+  ]);
```

Comment:
for `io` just pass in a `vstorageKit`. Then this can be:
```suggestion
  const [latestOutcome, latestQuestion] = await Promise.all([`${basePath}.latestOutcome`, `${basePath}.latestQuestion`].map(path => vsk.readPublished(path)););
```

be sure that `basePath` omits `published:`

after that I think this function doesn't do much and should just be inlined.